build-aux/rnote_inno.iss.in

```diff
@@ -0,0 +1,88 @@
+; Rnote Inno-Setup Installer Script
+
+#define MyAppName @APP_NAME_CAPITALIZED@
+#define MyAppVersion @APP_VERSION@
+#define MyAppPublisher @APP_AUTHOR_NAME@
+#define MyAppURL @APP_WEBSITE@
+#define MyAppExeName @APP_OUTPUT@
+#define MyAppAssocName MyAppName + " File"
+#define MyAppAssocExt ".rnote"
+#define MyAppAssocKey StringChange(MyAppAssocName, " ", "") + MyAppAssocExt
+
+#define app_id = @APP_ID@
+#define meson_source_root = @MESON_SOURCE_ROOT@
+#define meson_build_root = @MESON_BUILD_ROOT@
+#define meson_install_prefix @MESON_INSTALL_PREFIX@
+#define msys_path @MSYS_PATH@
+
+[Setup]
+; NOTE: The value of AppId uniquely identifies this application. Do not use the same AppId value in installers for other applications.
+; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
+AppId={{14CA6E6F-6384-467B-B139-D6305C0F4431}
```

Comment:
@flxzt is the second { intended?

Reply:
yes, in inno this is the "escape character" to write an actual opening curly brace instead of (attempting to) replace it by a constant: https://jrsoftware.org/ishelp/index.php?topic=consts . I believe this is also what the wizard generates as template